com.unity.cinemachine/Editor/Utility/CmPipelineComponentInspectorUtility.cs

```diff
@@ -62,7 +62,7 @@ public enum RequiredTargets { None, Tracking, LookAt, Group };
                             case RequiredTargets.Group: noTarget |= t.FollowTargetAsGroup == null; break;
                         }
                     }
-                    else
+                    else if (targets[i] is CinemachineExtension)
```

Comment:
Same here as above.

Can cast x directly:
`var x = (CinemachineExtension)targets[i];`
or
`else if (targets[i] is CinemachineExtension x)`